### DISCUSSION TITLE
Process for disabling Tailwind colors without affecting Daisy?

### DISCUSSION THREAD

**Discussion by danawoodman** [saadeghi/daisyui#2241] [Q&A] [answered]
I'm looking to remove all of Tailwind's automatic color names and only allow Daisy semantic names to be used/autocompleted.

I tried setting `corePlugins` to false for all the various color types but that broke Daisy.

Is there a canonical way to only allow Daisy semantic colors?

**Comment by saadeghi** [comment]
If you don't use Tailwind's color utility classes (like `bg-red-500`) the class name for them will not be generated for production anyway so there's no need to disable things.

Is it a situation where you have Tailwind CSS color utility classes in your markup and you want them to exist in your production as class names without any style?

**Reply by danawoodman** [reply]
Yes, I know that, but what I'm looking for is not to muddy up my auto-complete list of class names and preventing other devs from using these classes because they're not present anymore. I only want my editor and my team to see the Daisy specific semantic classes

**Reply by saadeghi** [reply, marked as answer]
@danawoodman I see... In that case, setting an empty object for the Tailwind CSS colors must work:
```js
// tailwind.config.js

module.exports = {
  //...
  theme: {
    colors: {},
  },
}
```

**Reply by danawoodman** [reply]
Hmm, I've tried that but no luck, but I'll keep poking at it, thanks @saadeghi

**Reply by saadeghi** [reply]
It should work. At least it's working for me on the latest version of VScode + the latest version of `Tailwind CSS IntelliSense` plugin:  

Without `colors: {}`

<img width="850" alt="1" src="https://github.com/saadeghi/daisyui/assets/7342023/d53f22da-f599-4ec0-801d-1baa1234e01f">

With `colors: {}`

<img width="850" alt="2" src="https://github.com/saadeghi/daisyui/assets/7342023/3ba5d238-d63b-40e2-aa2d-884b7eb1221e">

**Reply by danawoodman** [reply]
It could have something to do with caching in my monorepo or something. I'm gonna try it on another project to confirm. Thanks for investigating!
